workflows/bacterial_genomics/bacterial_genome_assembly/bacterial_genome_assembly.ga

```diff
@@ -0,0 +1,705 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Short paired-end read analysis to provide assembly quality metrics",
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "abromics-consortium",
+            "url": "https://www.abromics.fr/"
+        },
+        {
+            "class": "Person",
+            "email": "mailto:abromics-support@groupes.france-bioinformatique.fr",
+            "identifier": "",
+            "name": "Abromics"
+        },
+        {
+            "alternateName": "pimarin",
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0002-8304-138X",
+            "name": "Pierre Marin"
+        },
+        {
+            "alternateName": "clsiguret",
+            "class": "Person",
+            "identifier": "https://orcid.org/0009-0005-6140-0379",
+            "name": "Clea Siguret"
+        }
+    ],
+    "format-version": "0.1",
+    "release": "1.0",
+    "license": "GPL-3.0-or-later",
+    "name": "bacterial_genome_assembly",
+    "steps": {
+        "0": {
+            "annotation": "abromics,genomic,input,trimmed_reads,forward,fastq, paired R1 input reads",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "abromics,genomic,input,trimmed_reads,forward,fastq, paired R1 input reads",
+                    "name": "paired_r1"
+                }
+            ],
+            "label": "paired_r1",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 0,
+                "top": 29.199996948242188
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"format\": [\"fastq\", \"fastq.gz\", \"fastqsanger\", \"fastqsanger.gz\"], \"tag\": \"\"}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "b1c7fa52-2e6f-45b1-bd48-d20bfcf9eeaa",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "abromics,genomic,input,trimmed_reads,reverse,fastq, paired R1 input reads",
+            "content_id": null,
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "abromics,genomic,input,trimmed_reads,reverse,fastq, paired R1 input reads",
```

Comment:
Same as the comment above, but also note that you wrote paired R1 input reads here.

Generally speaking it would be better if the two inputs were instead a single paired collection input, so that sample identifiers are tracked automatically.

Reply:
I understand why you say that. But for now, I'll keep it like this. Maybe for a new version of the workflow :smile: